prow/config/config.yaml

```diff
@@ -1074,36 +1061,6 @@ tide:
         - do-not-merge/work-in-progress
         - needs-rebase
       reviewApprovedRequired: true
-    - repos:
-        - tikv/pd
-        - tikv/tikv
-      includedBranches: # release branches and hotFix branches
-        - release-5.4
-        - release-6. # release-6.x.y, release-6.x.y-*
-        - release-6.1 # LTS: avoid prow's "Merging to branch *** is forbidden" description issue.
-        - release-6.5 # LTS: avoid prow's "Merging to branch *** is forbidden" description issue.
-        - release-7. # release-7.x.y, release-7.x.y-*
-        - release-7.1 # LTS: avoid prow's "Merging to branch *** is forbidden" description issue.
-        - release-7.5 # LTS: avoid prow's "Merging to branch *** is forbidden" description issue.
-        - release-8. # release-8.x.y, release-8.x.y-*
-        - release-8.0 # DMR: avoid prow's "Merging to branch *** is forbidden" description issue.
-        - release-8.1 # LTS: avoid prow's "Merging to branch *** is forbidden" description issue.
-        # - release-{{.ver}}
-      labels:
-        - status/can-merge
-        # no need to query this label since
-        #   the `do-not-merge/cherry-pick-not-approved` label will be execluded.
-        # - cherry-pick-approved
-      missingLabels:
-        - do-not-merge/blocked-paths
-        - do-not-merge/cherry-pick-not-approved
-        - do-not-merge/hold
-        - do-not-merge/invalid-title
-        - do-not-merge/needs-linked-issue
-        - do-not-merge/needs-triage-completed
-        - do-not-merge/release-note-label-needed
-        - do-not-merge/work-in-progress
-        - needs-rebase
```

Comment:
Why is this part of the configuration no longer needed?

Reply:
It has already been merged into the following configuration, please ignore my comments.